content/en/docs/instrumentation/erlang/testing.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Testing"
+draft: true
+---
+
+When relying on OpenTelemetry for alerts, pages, or SLIs/SLOs, it can be important to test that certain spans are created and attributes correctly set.
+
+## Setup
+Only the `opentelemetry` and `opentelemetry_api` libraries are required for testing in Elixir/Erlang:
+
+{{< tabs Erlang Elixir >}}
+
+{{< tab >}}
+{deps, [{opentelemetry_api, "~> 1.0"},
+        {opentelemetry, "~> 1.0"}]}.
+{{< /tab >}}
+
+{{< tab >}}
+def deps do
+  [
+    {:opentelemetry_api, "~> 1.0"},
+    {:opentelemetry, "~> 1.0"}
+  ]
+end
+{{< /tab >}}
+
+{{< /tabs >}}
+
+The test configuration should set the `exporter` to `:undefined` and the `:otel_batch_processor`'s delay to a minimum.
+
+{{< tabs Erlang Elixir >}}
+
+{{< tab >}}
+TODO
+{{< /tab >}}
+
+{{< tab >}}
+# config/test.exs
+import Config
+
+config :opentelemetry,
+    traces_exporter: :undefined
+
+config :opentelemetry, :processors, [
+  {:otel_batch_processor, %{scheduled_delay_ms: 1}}
```

Comment:
```suggestion
  {:otel_simple_processor, %{}}
```

The simple processor will export the span immediately and since it'll just be sending to a PID it doesn't matter that it also blocks for each span when it ends.